backend/service/k8s/hpa.go

```diff
@@ -3,6 +3,7 @@ package k8s
 import (
 	"context"
 
+	"github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
since ptypes is deprecated not sure if we want to be using ptypes/wrappers

cc @danielhochman

Reply:
yeah the new package is wrapperspb if you want to swap it out. if not i'll catch it in my cleanup.